### PR TITLE
Migrate authorization service to MongoDB

### PR DESCRIPTION
**Pull request by MikolaLvivUA**

### CODE REVIEW

File: src/controllers/user/user.controller.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ class UserController {
     async createUser(req: Request, res: Response, next: NextFunction) {
         try {
             const user  = req.body;
+            console.log(user);
```
Comment (VictorKmin) [inline src/controllers/user/user.controller.ts]: rm

File: src/controllers/auth/auth.controller.ts [hunk 2]
```diff
@@ -1,19 +1,29 @@
 import { NextFunction, Response } from 'express';
 
+import { UserActionEnum } from '../../constants/enums';
+import { tokenizer } from '../../helpers';
+import { oauthService } from '../../services/oauth';
```
Comment (VictorKmin) [inline src/controllers/auth/auth.controller.ts]: ../../services
../../constants

File: src/database/models/oauth_token.schema.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { Document, Model, model, Schema, Types } from 'mongoose';
+import { config } from '../../configs';
```
Comment (VictorKmin) [inline src/database/models/oauth_token.schema.ts]: line

File: src/database/models/user.schema.ts [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    },
+    photo_path: String,
+    created_at: {
+        type: Date,
+        default: Date.now,
+        required: true
+    },
+    updated_at: {
+        type: Date,
+        default: Date.now
```
Comment (VictorKmin) [inline src/database/models/user.schema.ts]: no need a default value

File: src/middlewares/user/checkEmailPresent.ts [hunk 5]
```diff
@@ -8,7 +8,7 @@ export const checkIsEmailPresent = async (req: Request, res: Response, next: Nex
     try {
         const { email } = req.body;
         const isUserPresent = await userService.getUserByParams({ email });
-
+        console.log(isUserPresent);
```
Comment (VictorKmin) [inline src/middlewares/user/checkEmailPresent.ts]: rm

File: src/routes/auth/auth.router.ts [hunk 6]
```diff
@@ -1,12 +1,13 @@
 import { Router } from 'express';
 
 import { authController } from '../../controllers';
+import { checkIsPasswordCorrect, checkIsUserRegistered } from '../../middlewares/oauth';
```
Comment (VictorKmin) [inline src/routes/auth/auth.router.ts]: ../../middlewares

File: src/controllers/user/user.controller.ts (hunk 1 above)
Comment (MikolaLvivUA) [reply src/controllers/user/user.controller.ts]: fixed

File: src/controllers/auth/auth.controller.ts (hunk 2 above)
Comment (MikolaLvivUA) [reply src/controllers/auth/auth.controller.ts]: fixed

File: src/database/models/user.schema.ts (hunk 4 above)
Comment (MikolaLvivUA) [reply src/database/models/user.schema.ts]: fixed

File: src/routes/auth/auth.router.ts (hunk 6 above)
Comment (MikolaLvivUA) [reply src/routes/auth/auth.router.ts]: fixed

File: src/middlewares/user/checkEmailPresent.ts (hunk 5 above)
Comment (MikolaLvivUA) [reply src/middlewares/user/checkEmailPresent.ts]: fixed

File: src/database/models/oauth_token.schema.ts (hunk 3 above)
Comment (MikolaLvivUA) [reply src/database/models/oauth_token.schema.ts]: fixed

**Pull request merged**